sdcm/nemesis.py

```diff
@@ -1898,6 +1898,17 @@ def repair_nodetool_repair(self, node=None, publish_event=True):
         with adaptive_timeout(Operations.REPAIR, node, timeout=HOUR_IN_SEC * 48):
             node.run_nodetool(sub_cmd="repair", publish_event=publish_event)
 
+    def run_repair_on_nodes(self, nodes: list, publish_event=True):
+        """
+        Ignore errors so that a single failure doesn’t cause the entire nemesis failure.
+        """
```

Comment:
Need another docstring.
`Execute a nodetool repair on the specified nodes, disregarding errors that may arise from failed or unavailable nodes during the process.`